internal/worker/worker.go

```diff
@@ -411,7 +408,7 @@ RUN rm /kairos-init`
 }
 
 func buildOCI(contextDir, image string) string {
-	return fmt.Sprintf(`docker build %s -t %s`, contextDir, image)
+	return fmt.Sprintf(`docker buildx build --load --progress=plain %s -t %s`, contextDir, image)
```

Comment:
We were seeing some deprecation warning because of using `docker build`. This fixes it. `--progress=plain` is to print the logs without manipulating the terminal be replacing lines. `--load` means load the image to the docker daemon.